testinfra/modules/base.py

```diff
@@ -35,11 +35,11 @@ def run_expect(self, expected, command, *args, **kwargs):
         return out
 
     def run_test(self, command, *args, **kwargs):
-        """Run command and check it return an exit status of 0 or 1
+        """Run command and check it return an exit status of 0, 1 or 127
 
         :raises: AssertionError
         """
-        return self.run_expect([0, 1], command, *args, **kwargs)
+        return self.run_expect([0, 1, 127], command, *args, **kwargs)
```

Comment:
don't modify the behavior or `run_test()`, directly use `run_expect([0, 1, 127], "command -v ...")` where you need it instead.